libyul/AsmAnalysis.cpp

```diff
@@ -590,7 +590,6 @@ bool AsmAnalyzer::warnOnInstructions(evmasm::Instruction _instr, SourceLocation
 			"incorrect stack access. Because of that they are disallowed in strict assembly. "
 			"Use functions, \"switch\", \"if\" or \"for\" statements instead."
 		);
-		m_success = false;
 	}
```

Comment:
After removing this line, braces are no longer needed here. 

BTW, They're also not needed in the `if` branches above, though it's outside of the code modified by this PR.